### PR TITLE
Geocoder library improvements.

### PR DESCRIPTION
**Pull request by DevX86**
Supports CI/CD.
Better error support for logging.

### CODE REVIEW

File: geocoder.go [hunk 1]
```diff
@@ -12,13 +12,11 @@ import (
...
-// The user should set the API KEY provided by Google
+// ApiKey The user should set the API KEY provided by Google
 var ApiKey string
 
-// Define the Geocode API URL as a constant
-const (
-	geocodeApiUrl = "https://maps.googleapis.com/maps/api/geocode/json?"
-)
+// GeocodeApiUrl The user should set the API URL with the default value being provided by Google.
+var GeocodeApiUrl = "https://maps.googleapis.com/maps/api/geocode/json?"
```
Comment (kelvins) [inline geocoder.go]: I think it's ok to rename to `ApiUrl` as you suggested [here](https://github.com/kelvins/geocoder/issues/6).

**Review by kelvins — COMMENTED** [review]
Nice, lgtm, just left a comment related to your suggestion. 🚀

File: geocoder.go (hunk 1 above)
Comment (DevX86) [reply geocoder.go]: Sorry for the late response, sounds good! I'll get this updated.

**Comment by DevX86** [comment]
@kelvins Updated!

**Comment by DevX86** [comment]
@kelvins Bump.

**Review by kelvins — APPROVED** [review]
Nice, thanks a lot @DevX86!

**Pull request merged**